nsist/pypi.py

```diff
@@ -181,7 +182,8 @@ def extract_wheel(whl_file, target_dir):
     for p in td.iterdir():
         if p.suffix not in {'.data', '.dist-info'}:
             if p.is_dir():
-                shutil.copytree(str(p), str(target / p.name))
+                # If the dst directory already exists, this will combine them
```

Comment:
Can the comment explicitly say that this is why we use this rather than shutil.copytree? Thanks.